src/database/models/user.py

```
@@ -0,0 +1,36 @@
+from typing import Optional, List, Dict, Any
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import

(F401)